test/utils_test.js

```diff
@@ -603,4 +627,23 @@ describe("utils", () => {
       expect(pad(4, 3)).eql("004");
     });
   });
+
+  describe("toErrorList()", () => {
+    it("should convert an errorSchema into a flat list", () => {
+      expect(toErrorList({
+        a: {
+          b: {
+            __errors: ["err1", "err2"]
+          }
+        },
+        c: {
+          __errors: ["err3"]
+        }
+      })).eql([
+        {stack: "b err1"},
+        {stack: "b err2"},
```

Comment:
Well, this output format no longer corresponds to the `jsonschema` return type. Specifically, in `jsonschema` it would be `instance.a.b`, and there would be properties called `property`, `name`, `instance`, etc. I think we should either match that output or document the format we're producing instead. I don't have a very clear sense of what the use case is for the error list, so I don't know which is better. It looks like you intend for `stack` to be usable directly as an error message, so maybe we should call it `errorMsg`. I think we will probably find that having the path available is going to be useful eventually even if it doesn't show up in the error message (that's why `jsonschema` provides it).